src/lib/support/jsontlv/JsonToTlv.h

```diff
@@ -39,4 +39,11 @@ CHIP_ERROR JsonToTlv(const std::string & jsonString, TLV::TLVWriter & writer);
  */
 CHIP_ERROR ConvertTlvTag(const uint64_t tagNumber, TLV::Tag & tag);
 
+/*
+ * Given a JSON object that represents TLV, this function remove outer structure container, and writes the corresponding TLV bytes
+ * into the provided buffer. The size of tlv will be adjusted to the size of the actual data written to the buffer.
+ * For example, the tlv blob consumed by IM write needs the one without container, the JsonToTlv expect Json input has the struct,
+ * this function can help to generate tlv without struct.
```

Comment:
I am having a very hard time understanding this comment, partly because of the grammar and partly because I don't understand what it's producing at the end (sounds like partial TLV?).  An example of what API would consume the resulting ByteSpan might help me understand what this comment should actually say.

Reply:
updated, thanks